spec/integration/_dev/benchmarks/scenario.spec.yml

```
@@ -0,0 +1,30 @@
+spec:
+  type: object
+  additionalProperties: false
+  required:
+    - version
+    - data_stream
+    - description
+    - corpora
+  properties:
+    version:
+      $ref: "../integration/manifest.spec.yml#/definitions/version"
```

This relative path for this reference should be:

```suggestion
      $ref: "../../manifest.spec.yml#/definitions/version"
```